app.js

```diff
@@ -2,87 +2,75 @@ const storageKey = 'awesomeBooks';
 const currentFormTitle = document.querySelector('#title');
 const currentFormAuthor = document.querySelector('#author');
 const currentFormBookAddButton = document.querySelector('#book-add-button');
-const arrBooks = [];
 
-function removeBook(id) {
-  for (let i = 0; i < arrBooks.length; i += 1) {
-    if (arrBooks[i].id === id) {
-      arrBooks.splice(i, 1);
-    }
+class BookShelf {
+  constructor() {
+    this.arrBooks = [];
   }
-}
-
-function addBook() {
-  let maxId = 0;
-  arrBooks.forEach((book) => {
-    if (book.id > maxId) {
-      maxId = book.id;
-    }
-  });
 
-  const newBook = {
-    id: maxId + 1,
-    title: currentFormTitle.value,
-    author: currentFormAuthor.value,
-  };
-  arrBooks.push(newBook);
-}
+  addBook(newTitle, newAuthor) {
+    const timeNow = new Date();
+    const newBook = {
+      id: timeNow.getTime(),
+      title: newTitle,
+      author: newAuthor,
+    };
 
```

Comment:
- [ ] I am unable to add a book. After populating inputs and clicking the add button, the app does not save the book on the local storage and neither does it present it on the homepage. Kindly fix this.

  ![absg](https://user-images.githubusercontent.com/81468675/199455288-cbf57b3a-369c-4535-9b1e-ac5503be35c2.jpg)